auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailFragment.java

```diff
@@ -210,6 +213,39 @@ public void onClick(View view) {
         });
     }
 
+    private void setUpPrivacyPolicy() {
+        if (TextUtils.isEmpty(mHelper.getFlowParams().privacyPolicyUrl)) {
+            return;
+        }
+
+        ForegroundColorSpan foregroundColorSpan =
+                new ForegroundColorSpan(ContextCompat.getColor(getContext(), R.color.linkColor));
+
+        String link = getString(R.string.privacy_policy);
+        SpannableStringBuilder spannableStringBuilder = new SpannableStringBuilder(link);
+        int start = mAgreementText.length();
+        spannableStringBuilder.setSpan(foregroundColorSpan, start, start + link.length(), 0);
+
+        mAgreementText.append(" and the ");
+        mAgreementText.append(spannableStringBuilder);
+        mAgreementText.setOnClickListener(new View.OnClickListener() {
```

Comment:
I don't think this will work. Try clicking the Terms of Service link and it will open the privacy policy instead. Oops! 😄 You'll need to make only the bits of text clickable instead of the entire view like so: https://stackoverflow.com/a/10697453/4548500.